python/ray/data/_internal/logging.yaml

```diff
@@ -4,16 +4,25 @@ disable_existing_loggers: False
 formatters:
     ray:
         format: "%(asctime)s\t%(levelname)s %(filename)s:%(lineno)s -- %(message)s"
+    ray_json:
+        class: ray._private.ray_logging.formatters.JSONFormatter
 
 filters:
     console_filter:
         (): ray.data._internal.logging.HiddenRecordFilter
+    core_context_filter:
+        class: ray._private.ray_logging.filters.CoreContextFilter
```

Comment:
`(): ray._private.ray_logging.filters.CoreContextFilter` will enable this core context filter, I think.